notebooks/README.md

```diff
@@ -0,0 +1,10 @@
+# Growth Model Notebooks
+
+The file contains the notebooks used for developing growth model. The following list describing each of them. 
```

Comment:
maybe reword it to: `This directory contains the Jupyter notebooks used.... The description for each of them are as follows:`

Reply:
We can also remove the `.` at the end of each description